string jointName
float64 angularVelocity
float64 max
